spec/tree_spec.rb

```diff
@@ -1,14 +1,175 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe Tree do
+  let(:tree) { Tree.new }
+
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(described_class).to be_a(Class)
+  end
+
+  it 'can age to a positive, non-zero maximum' do
+    expect { Tree.new(-1) }.to raise_error(ArgumentError)
+    expect { Tree.new(0) }.to raise_error(ArgumentError)
+
+    expect { Tree.new(1) }.not_to raise_error
+    expect { Tree.new }.not_to raise_error
+  end
+
+  it 'can grow by a positive, non-zero rate' do
+    expect { Tree.new(max_growth: -1) }.to raise_error(ArgumentError)
+    expect { Tree.new(max_growth: 0) }.to raise_error(ArgumentError)
+
+    expect { Tree.new(max_growth: 1) }.not_to raise_error
+    expect { Tree.new }.not_to raise_error
+  end
+
+  it 'starts out alive' do
+    expect(tree).not_to be_dead
+  end
+
+  it 'starts out with no apples' do
```

Comment:
I love the story approach of your tests. Keep in mind using contexts to improve readability.

www.betterspecs.org/#contexts